docs/create/creating-an-xrpk.md

```diff
@@ -0,0 +1,79 @@
+---
+id: creating-an-xrpk
+title: Create An XRPackage
+---
+
+This guide will walk you through the basic steps for creating an XRPackage for use in the Webaverse ecosystem.
+
+The term "XRPK" stands for "XRPackage", which can be thought of as an "application" or "widget" in the Webaverse. 
+
+XRPackages in the Webaverse are spatialized apps. Unlike traditional apps that exists on a 2D plane, XRPKs are applications that exist in three dimensional space.
+
+The `xrpk` command line tool will be used to package the assets into a bundle. Although XRPackage supports multiple asset types, this guide demonstrates packaging an existing ThreeJS 3d website scene.
+
+**Note**: This guide assumes you have read the [XRPackage Overview](./xrpackage-overview.md) or are familiar with what an XRPackage is.
+
+## Prerequisites
+
+- Familiarity with the command line
+- NodeJS and NPM installed
+
+## Install XRPackage
+
+The first step is to install the <a href="https://www.npmjs.com/package/xrpk" target="_blank" rel="noopener noreferrer">XRPackage CLI</a>. This tool is used to create and test XRPackages.
+
+```bash
+$ npm install -g xrpk
+```
+
+## Clone The Project Repository
+
+For the purposes of this guide, a barebones 3D website scene built with ThreeJS is available at the repository below. This scene contains a simple cube with physics.
+
+```bash
+$ git clone https://github.com/chrislatorres/physicscube.git
+```
+
+You can also follow this guide to package an existing WebXR scene. If you are packaging an existing WebXR site, consider making the scene's skybox transparent so that it can be composited along other XRPKs and the base reality layer. 
+
+Additionally, be mindful of bundle sizes: excessively large bundles will result in a delayed loading experience for users. Try to stay below 100mb.
+
+## Create The Package Manifest
+
+When building an XRPackage with the CLI, a `manifest.json` file in the root directory will be read to define the package's configuration. The sample below is the simplest possible manifest file. Copy the contents below into a file named `manifest.json` in the root directory of the package, or use the CLI `xrpk init` command to generate it for you:
+
+```json
+{
+  "name": "physicscube",
+  "description": "Physics Cube XRPackage",
+  "xr_type": "webxr-site@0.0.1",
```

Comment:
`xr_type` is not needed right now. If the software blocks this otherwise, we should probably get rid of that block.

Reply:
62918aa